.circleci/config.yml

```diff
@@ -794,8 +855,17 @@ workflows:
           <<: *filter-pr-branch
   e2etests:
```

Comment:
Weird to put this into the e2e workflow, but jobs cannot depend on other jobs outside of their workflow.